app/views/queue_items/index.html.haml

```diff
@@ -0,0 +1,41 @@
+%section.my_queue.container
+  .row
+    .col-sm-10.col-sm-offset-1
+      %article
+        %header
+          %h2 My Queue
+        %table.table
+          %thead
+            %tr
+              %th(width="10%") List Order
+              %th(width="30%") Video Title
+              %th(width="10%") Play
+              %th(width="20%") Rating
+              %th(width="15%") Genre
+              %th(width="15%") Remove
+          %tbody
+            - @my_queue.each do |queue_item|
+              %tr
+                %td
+                  %input.form-control(type="text" value="#{queue_item.display_order}")
+                %td
+                  = link_to queue_item.video.title, queue_item.video
+                %td
+                  = button_to "Play", nil, class: "btn btn-default"
+                %td
+                  = queue_item.rating(current_user)
+
+                  / %select :rating, options_for_select([5,4,3,2,1].map { |number| [pluralize(number, "Star")]})
+
+                  / %select.form-group(name="", value="3")
+                  /   %option(value="5") 5 Stars
+                  /   %option(value="4") 4 Stars
+                  /   %option(value="3") 3 Stars
+                  /   %option(value="2") 2 Stars
+                  /   %option(value="1") 1 Star
```

Comment:
Don't leave commented code, especially in your view templates. They are translated to html comments, which are viewable by the public in the html source.